### ISSUE TITLE
Feature Request: different config for multistage transitions

### ISSUE THREAD

**Issue by Nizarius** [react-spring/react-spring#629]
Is it possible to pass different config in next's object parameter? 
```
leave: item => async (next, cancel) => {
    await next({ life: '0%' })
    await next({ opacity: 0 })
    await next({ height: 0 })
  },
```
I really need this functionality to make one step's duration 0ms (for out of screen elements).

If not, I want to make PR request for adding this functionality. Please, let me know.

**Comment by aleclarson** [comment]
This is implemented by #615 which will be merged soon

**Comment by Nizarius** [comment]
@aleclarson great! Thank you.
